### PR TITLE
Add projection and region to grdview docstring

### PR DESCRIPTION
**Pull request by meghanrjones**
**Description of proposed changes**

The `projection` parameter is included in the list of aliases but not the docstring for grdview. This PR adds the projection description to the docstring.

Fixes #

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

**Review by weiji14 — APPROVED** [review]

File: pygmt/src/grdview.py [hunk 1]
```diff
@@ -58,6 +58,7 @@ def grdview(self, grid, **kwargs):
     grid : str or xarray.DataArray
         The file name of the input relief grid or the grid loaded as a
         DataArray.
+    {J}
```
Comment (weiji14) [inline pygmt/src/grdview.py:61]:
```suggestion
    {R}
    {J}
```

**Review by weiji14 — CHANGES_REQUESTED** [review]
Oh wait, actually, we're missing the region (R) docstring as well for this!

File: pygmt/src/grdview.py (hunk 1 above)
Comment (meghanrjones) [reply pygmt/src/grdview.py:62]: Good catch, thanks

**Review by weiji14 — APPROVED** [review]
So region (-R) isn't actually a required argument for `grdview` according to https://docs.generic-mapping-tools.org/6.2.0rc2/grdview.html, but using `{R}` in the docstring here says that it is:

image

Wondering if we should edit the wording slightly at https://github.com/GenericMappingTools/pygmt/blob/01e2febb77f08370cb8ef8e23f7b1c42e9709c99/pygmt/helpers/decorators.py#L17-L21

Can open a separate issue/PR for this.

**Comment by meghanrjones** [comment]
> 
> Wondering if we should edit the wording slightly
> 


I think the current wording is accurate for most plotting modules, so maybe I can just not use the replacement text for `grdview` but keep the wording from decorators.py as is to not affect other method docs?

**Comment by meghanrjones** [comment]
Is the latest version OK with you, @weiji14?

**Comment by weiji14** [comment]
> Is the latest version OK with you, @weiji14?

Yep, looks good! I like that it explicitly specifies zmin/zmax too. Remember to change the PR title to include 'region' before merging.

**Comment by meghanrjones** [comment]
Sounds good. One more question - when the checks fail for reasons clearly unrelated to the PR (like now), do you prefer for me to re-run all checks or admin merge the PR?

**Comment by weiji14** [comment]
> when the checks fail for reasons clearly unrelated to the PR (like now), do you prefer for me to re-run all checks or admin merge the PR?

I would just admin merge in this case to conserve CI resources.

**Pull request merged**